aten/src/ATen/test/complex_test.cpp

```diff
@@ -0,0 +1,34 @@
+#include <gtest/gtest.h>
+#include "c10/util/Complex.h"
+
+template<typename T, typename iT>
+static void TestIntegerOp(T real, T img, iT i) {
```

Comment:
this could be change to:
```
template<typename T, typename int_t>
TestBinaryOps(T real, T imag, int_t num) {
...
}
```
I think having the int numbers represent by it makes it a little hard to read the code since we are dealing with complex values :D

Reply:
I agree. Updated